.azure/app-cloud-e2e.yml

```diff
@@ -148,11 +148,11 @@ jobs:
     # It also e2e tests running on cloud without installing dependencies.
     - bash: |
         git clone https://github.com/Lightning-AI/lightning-quick-start examples/app/quick-start
-      condition: eq(variables['name'], 'quick_start')
+      condition: and(succeeded(), eq(variables['name'], 'quick_start'))
```

Comment:
from https://learn.microsoft.com/en-us/azure/devops/pipelines/process/conditions

> When you specify your own condition property for a stage / job / step, you overwrite its default condition: succeeded(). This can lead to your stage / job / step running even if the build is cancelled. Make sure you take into account the state of the parent stage / job when writing your own conditions.

Reply:
You need to add this comment to each of your chabegs

Reply:
Done